docs/_articles/en/soql/soql-builder.md

```diff
@@ -12,6 +12,7 @@ During beta, you can build simple query statements that include:
 - FROM clause for only one sObject type
 - SELECT clause to pick fields from the selected sObject
 - WHERE clause to filter your data
+- COUNT clause to return the number of rows that match the filtering conditions <!-- ekapner, 2/12: new docs -->
```

Comment:
COUNT is not a clause. It is an aggregate function.
See more info here: https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_count.htm

In the SOQL builder UI we only support one out of the 2 possible ways we could use COUNT. The one we support is COUNT() which shows up in the list of fields that we can pick in the SELECT clause.
Also, you don't need a filtering condition to use COUNT.

Maybe, the best place to document this could be in line 13, where you could say

"SELECT clause to pick fields from the selected sObject or pick COUNT() to perform an aggregation of the results"

Reply:
+1